bats/tests/helpers/vm.bash

```diff
@@ -320,9 +325,45 @@ docker_context_exists() {
     assert_output "$RD_DOCKER_CONTEXT"
 }
 
+# Check if the VM is using systemd (instead of OpenRC).
+using_systemd() {
+    [[ -n ${RD_USING_SYSTEMD:-} ]] || load_var RD_USING_SYSTEMD || true
```

Comment:
Why do we use a leading underscore in `_RD_SUPPORTS_VZ_EMULATION` but not here?

Reply:
Because I wrote them at different points in time. Underscore prefix added.

My current thinking is, `RD_*` should be things user might set, `_RD_` is for caches of detected things?